includes/abbreviations.md

```diff
@@ -25,7 +25,8 @@
 *[SRI]: Subresource Integrity
 *[SSH]: Secure Shell
 *[SSL]: Secure Sockets Layer
+*[the Mend Renovate App]: this means the Mend-hosted GitHub app
```

Comment:
I'm marking this as _draft_, because I first need to do resolve this upstream issue:

- https://github.com/renovatebot/renovate/issues/23333